app/style/SideBar.less

```diff
@@ -105,25 +113,53 @@
 }
 
 .sidebar-menu-total-balance-extended-bottom-account-number {
-  font-family: var(--font-family-monospace);
+  font-family: var(--font-family-regular);
   color: var(--input-color-default);
   font-size: 15px;
+
+  span {
+    font-size: 15px;
```

Comment:
This is the offending rule in my case. Commenting this makes the sizes display correctly.

Weird it's not failing for you. Which OS are you using?

Reply:
I use macOS Catalina. If I remove the suggested lines, then the fonts are too small: 
<img width="303" alt="Screenshot 2019-11-19 at 21 35 46" src="https://user-images.githubusercontent.com/52497040/69184159-99742a80-0b14-11ea-95da-f09867df6ad7.png">


In the design specs, the bold and the regular part of the text are 15px too. 
I think in your case the normal part is good and the bold is too small somehow.
I will try to reproduce this on Linux.

Reply:
Sorry, but I could not reproduce it on Linux either. Maybe it is because I run it in a virtual machine.
<img width="303" alt="Screenshot 2019-11-20 at 11 31 02" src="https://user-images.githubusercontent.com/52497040/69231382-45575d80-0b89-11ea-9929-7e680e2a3144.png">